File: odd-number/golang/raw.go
package main

import (
	"fmt"
	"time"
	"math"
)

func oddNumbers(l int, r int) int {

	fmt.Println("Running:")

	tot := 0

	for n := l ; n <= r ; n += 1 {
		isOdd := true
		fmt.Println("\t",n)

		for i := 2; i < n; i += 1 { // test only inpar numbers
			if n % i == 0 {
					isOdd = false
			}
		}

		if isOdd {
			tot++
		}

	}

	return tot
}

func main ( ) {

	start := time.Now()
	fmt.Println("Start")
	result := oddNumbers(2, int(math.Pow(10, 6.0)))
	end := time.Now()
	fmt.Println("End")
	fmt.Println("Tot:", result)
	fmt.Println("Ele:", end.Sub(start))

}

File: odd-number/php/raw.php
<?php

function oddNumbers($l, $r) {

    $tot = 0;

    echo "runing:" . PHP_EOL;
    for( $n = $l ; $n <= $r ; $n++ ){

        $isOdd = true;
        echo "\t$n" . PHP_EOL;

        for ( $i = 3; $i < $n; $i++ ) { // test only inpar numbers
            if ( $n % $i === 0 )
                $isOdd = false;
        }

        if($isOdd)
            $tot++;

    }

    return $tot;

}

$start = strtotime('now');
echo 'start: ' . date('Y-m-d H:i:s') . PHP_EOL;
$oddsCount = oddNumbers(1, 10 ** 6 );
echo 'end  : ' . date('Y-m-d H:i:s') . PHP_EOL;
echo 'total: ' . $oddsCount . PHP_EOL;
echo 'time : ' . round(((strtotime('now') - $start)/60) . 'm') . PHP_EOL;
